spiceaidocs/docs/components/models/filesystem.md

```diff
@@ -50,3 +50,20 @@ models:
       - path: models/llms/llama3.2-1b-instruct/tokenizer_config.json
       - path: models/llms/llama3.2-1b-instruct/config.json
 ```
+
+### Example: Overriding the Chat Template
```

Comment:
This requires some more explanation. I assume this is a Jinja format? What variables/functions are available? Is there a link to a language spec that explains how the language syntax works?